packages/compiler-cli/linker/babel/test/ast/babel_ast_host_spec.ts

```diff
@@ -96,6 +96,11 @@ describe('BabelAstHost', () => {
       expect(host.isBooleanLiteral(expr('false'))).toBe(true);
     });
 
+    it('should return true if the expression is a minified boolean literal', () => {
+      expect(host.isBooleanLiteral(expr('!0'))).toBe(true);
+      expect(host.isBooleanLiteral(expr('!1'))).toBe(true);
```

Comment:
Should we have falsy results as well?

```suggestion
      expect(host.isBooleanLiteral(expr('!1'))).toBe(true);
      expect(host.isBooleanLiteral(expr('!2'))).toBe(false);
      expect(host.isBooleanLiteral(expr('~1'))).toBe(false);
```

same for the parse test below and for the TS host.

Reply:
Added checks to the `it('should return false if the expression is not a boolean literal'` test.
I don't think there is any benefit in adding to the `it('should error if the value is not a boolean literal'`.